gravatar-ui/src/main/java/com/gravatar/ui/components/atomic/ViewProfileButton.kt

```diff
@@ -52,9 +54,10 @@ public fun ViewProfileButton(
     modifier: Modifier = Modifier,
     textStyle: TextStyle = MaterialTheme.typography.titleSmall.copy(color = MaterialTheme.colorScheme.onBackground),
     inlineContent: @Composable ((String) -> Unit)? = { DefaultInlineContent(textStyle.color) },
+    buttonText: String = stringResource(R.string.view_profile_button),
```

Comment:
I believe we should keep the @Composable parameter as the last one to keep the trailing lambda funtionality. 
Reordering will not be backward compatible so I guess we need to add a new `ViewProfileButton` with new params, thoughts?

Reply:
Nice suggestion. Yeah, I agree it's better to keep the `inlineContent` as the last parameter. 

I've pushed some changes. Let me know what you think.